docker/start.sh

```diff
@@ -43,6 +43,7 @@ setids() {
     PGID=${PGID:-1000}
     groupmod -o -g "$PGID" git
     usermod -o -u "$PUID" git
+    export USER=git
```

Comment:
We can go one step further (not sure if it will work, looks like so):
```suggestion
    export USER=git
    groupmod -o -g "$PGID" $USER
    usermod -o -u "$PUID" $USER
```